explorer/explorerroutes.go

```diff
@@ -319,8 +319,18 @@ func (exp *explorerUI) TxPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Initially try to get the transaction information from the "lite" mode
-	// data source, which relies on RPCs to dcrd.
+	inout, ok := r.Context().Value(ctxTxInOut).(string)
+	if !ok {
+		inout = ""
```

Comment:
If this happened, that would indicate a problem with our `TransactionIoIndexCtx` middleware, not the user input, so I think we should log this.  Also, on a failed type assertion with `!ok` being the case, the value of `inout` will be the string zero value (empty string).

Reply:
!ok is true if the user links to a transaction without any in/out designation.  as an example: `tx/1c5c53d3a604f4b875173e4681e3e789faeed1dd563e04970d1e5655c8cc885e`

So its not something we should log.  Its normal operation given we are sharing the route

Good to know on the empty string.  I'll try removing the entire !ok if statement.